### PR TITLE
♻️ reroute `get project inactivity` via `dynamic-scheduler`

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

Rerouted `get project inactivity` command via `dynamic-scheduler`.

## Related issue/s

- https://github.com/ITISFoundation/osparc-simcore/issues/5632
- https://github.com/ITISFoundation/osparc-simcore/issues/6894
- closes https://github.com/ITISFoundation/osparc-simcore/issues/6950

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by giancarloromeo — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

File: packages/models-library/src/models_library/api_schemas_directorv2/dynamic_services.py [hunk 1]
```diff
@@ -79,3 +79,5 @@ class DynamicServiceCreate(ServiceDetails):
 
 class GetProjectInactivityResponse(BaseModel):
     is_inactive: bool
+
+    model_config = ConfigDict(json_schema_extra={"example": {"is_inactive": "false"}})
```
Comment (pcrespov) [inline packages/models-library/src/models_library/api_schemas_directorv2/dynamic_services.py:83]: I wonder if these type of examples are really of any use :-)

**Review by pcrespov — COMMENTED** [review]
Basically the same question as in https://github.com/ITISFoundation/osparc-simcore/pull/6943#pullrequestreview-2496106386

**Review by sanderegg — APPROVED** [review]
thanks

**Review by pcrespov — APPROVED** [review]
thx for the explanation.

**Pull request merged**